From 2529049f56361da5d5349aa609fff0d0a4c5bb46 Mon Sep 17 00:00:00 2001
From: Ting-Ying Li <tingying.li@cypress.com>
Date: Thu, 15 Jul 2021 05:26:23 -0500
Subject: [PATCH 087/179] brcmfmac: Add SDIO verdor/device id for CYW89459 in
 header file

- Add more SDIO vendor and device ids for supporting CYW89459.
- Rename 89359 to 89459 for device id 0x4355.

Signed-off-by: Ting-Ying Li <tingying.li@infineon.com>
---
 include/linux/mmc/sdio_ids.h | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/include/linux/mmc/sdio_ids.h b/include/linux/mmc/sdio_ids.h
index 657b8005f082..64d43a61249c 100644
--- a/include/linux/mmc/sdio_ids.h
+++ b/include/linux/mmc/sdio_ids.h
@@ -63,7 +63,7 @@
 #define SDIO_DEVICE_ID_BROADCOM_4339		0x4339
 #define SDIO_DEVICE_ID_BROADCOM_4345		0x4345
 #define SDIO_DEVICE_ID_BROADCOM_4354		0x4354
-#define SDIO_DEVICE_ID_BROADCOM_CYPRESS_89359	0x4355
+#define SDIO_DEVICE_ID_BROADCOM_CYPRESS_89459	0x4355
 #define SDIO_DEVICE_ID_BROADCOM_4356		0x4356
 #define SDIO_DEVICE_ID_BROADCOM_4359		0x4359
 #define SDIO_DEVICE_ID_BROADCOM_CYPRESS_43439	0xa9af
@@ -78,8 +78,11 @@
 #define SDIO_DEVICE_ID_BROADCOM_43455		0xa9bf
 #define SDIO_DEVICE_ID_BROADCOM_CYPRESS_43752	0xaae8
 
-#define SDIO_VENDOR_ID_CYPRESS                  0x04b4
-#define SDIO_DEVICE_ID_CYPRESS_43439            0xbd3d
+#define SDIO_VENDOR_ID_CYPRESS			0x04b4
+#define SDIO_DEVICE_ID_CYPRESS_54590		0xbd3a
+#define SDIO_DEVICE_ID_CYPRESS_54591		0xbd3b
+#define SDIO_DEVICE_ID_CYPRESS_54594		0xbd3c
+#define SDIO_DEVICE_ID_CYPRESS_43439		0xbd3d
 
 #define SDIO_VENDOR_ID_MARVELL			0x02df
 #define SDIO_DEVICE_ID_MARVELL_LIBERTAS		0x9103
-- 
2.17.1

